### PR TITLE
Port to AdwAboutWindow

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
Closes https://github.com/sonnyp/Workbench/issues/142.

### CODE REVIEW

**Comment by TheEvilSkeleton** [comment]
This is how it looks like:
https://user-images.githubusercontent.com/50847364/189408571-9e7f7d14-8c41-4e67-ab09-0c95ee99bcdc.mp4

File: src/about.js [hunk 1]
```diff
@@ -25,32 +25,26 @@ ${getValaVersion()}
...
 `.trim();
 
-  const dialog = new Gtk.AboutDialog({
-    application,
-    authors: ["Sonny Piers https://sonny.re"],
-    comments: _("Learn and prototype with\nGNOME technologies"),
+  const dialog = new Adw.AboutWindow({
+    application_name: "Workbench",
+    developer_name: "Sonny Piers",
+    comments: _("Learn and prototype with GNOME technologies"),
```
Comment (sonnyp) [inline src/about.js:31]: if we remove "comments" - I think the "details" row is replaced with the website row directly which I would prefer

File: re.sonny.Workbench.json [hunk 2]
```diff
@@ -55,6 +55,46 @@
...
+          "sources": [
+            {
+              "type": "git",
+              "url": "https://github.com/lazka/sassc.git",
+              "branch": "meson"
+            }
+          ]
+        }
+      ]
+    },
```
[the remark this answers is not in the document]
Comment (TheEvilSkeleton) [reply re.sonny.Workbench.json]: Thanks, I hadn't thought of that.

File: src/about.js (hunk 1 above)
Comment (TheEvilSkeleton) [reply src/about.js]: Correct.

File: src/about.js [hunk 3]
```diff
@@ -25,32 +25,38 @@ ${getValaVersion()}
...
+    ],
+    designers: ["Tobias Bernard <tbernard@gnome.org>"],
   });
-  dialog.add_credit_section("Contributors", [
-    "Lorenz Wildberg https://gitlab.gnome.org/lwildberg",
-    "Tobias Bernard <tbernard@gnome.org>",
+
+  dialog.add_credit_section(_("Contributors"), [
     "Ben Foote http://www.bengineeri.ng",
+    "Hari Rana https://github.com/TheEvilSkeleton",
```
Comment (sonnyp) [inline src/about.js:51]: @TheEvilSkeleton are you happy with this? feel free to suggest something else

**Review by sonnyp — APPROVED** [review]
Thanks

`translator_credits` was missing but I added it back in https://github.com/sonnyp/Workbench/pull/150/commits/cccfba7fd45699d8cf91573b4b12d17b824d9138

File: src/about.js (hunk 3 above)
Comment (TheEvilSkeleton) [reply src/about.js:51]: Is `Hari Rana (TheEvilSkeleton) https://theevilskeleton.gitlab.io` good for you? I like to have my alias there.

**Comment by TheEvilSkeleton** [comment]
Looks good to me now. :)

**Comment by TheEvilSkeleton** [comment]
One last thing: after clicking on a link, I get this warning:
```
GVFS-Warning: The peer-to-peer connection failed: Error when getting information for file “/run/user/1002/gvfsd”: No such file or directory. Falling back to the session bus. Your application is probably missing --filesystem=xdg-run/gvfsd privileges.
```

Should I add that permission?

**Comment by sonnyp** [comment]
Probably not https://github.com/flathub/flathub/issues/2180

**Comment by TheEvilSkeleton** [comment]
Alright. It should be good to go!

**Pull request merged**